codecov.yaml

```diff
@@ -1,5 +1,5 @@
 codecov:
-  token: ${{secrets.CODECOV_TOKEN}}
+  token: 61ef0651-a3c3-4b71-8ca6-f82630f39579
```

Comment:
A clarification: does Antrea use the hardcoded token as well?

Reply:
Yes, this is what Antrea does: https://github.com/vmware-tanzu/antrea/blob/561d179f2e497fa834608c28705ac1e870cf3fc7/codecov.yaml#L2.